src/screens/add-user/add-user-page.tsx

```diff
@@ -23,8 +28,9 @@ export const AddUserPage = (props: NavigationComponentProps) => {
   const [birthDate, setBirthDate] = useState('');
   const [email, setEmail] = useState('');
   const [role, setRole] = useState('user');
-  const [errorMessage, setErrorMessage] = useState(['', false]);
-  const [createUser, {loading}] = useMutation(ADD_USER_MUTATION);
+  const [errorMessage, setErrorMessage] = useState(['', true]);
+  const [createUser, {loading, error}] = useMutation(ADD_USER_MUTATION);
+  const [validateInput, setValidateInput] = useState(true);
```

Comment:
```suggestion
  const [isInputValid, setIsInputValid] = useState(true);
```

Estados booleanos é legal nomearmos como se fosse uma pergunta tbm.